src/hashesorg.py

```diff
@@ -0,0 +1,26 @@
+def HashesOrg(hashed, key):
+    try:
+        output = requests.get(f"https://hashes.org/api.php?key={key}&query={hashed}", timeout=5).text
+    except:
+        print("Couldn't connect")

```

Comment:
Printing is bad, we should use a logging library like https://github.com/Delgan/loguru -- we should keep the terminal as clean as possible and rely on a printing class to print things for us :)